gio/src/unix_mount_entry.rs

```diff
@@ -9,30 +9,28 @@ use std::mem;
 
 impl UnixMountEntry {
     #[doc(alias = "g_unix_mount_at")]
-    pub fn new_at<P: AsRef<std::path::Path>>(mount_path: P) -> (UnixMountEntry, u64) {
+    pub fn new_at<P: AsRef<std::path::Path>>(mount_path: P) -> Option<(UnixMountEntry, u64)> {
         unsafe {
             let mut time_read = mem::MaybeUninit::uninit();
-            let ret = from_glib_full(ffi::g_unix_mount_at(
+            let ret: Option<UnixMountEntry> = from_glib_none(ffi::g_unix_mount_at(
                 mount_path.as_ref().to_glib_none().0,
                 time_read.as_mut_ptr(),
             ));
-            let time_read = time_read.assume_init();
-            (ret, time_read)
+            ret.map(|x| (x, time_read.assume_init()))
```

Comment:
Actually `time_read` is always valid, even if `NULL` is returned.

Reply:
Okay, was to lazy to check the C code